src/containers/MenuPage/MenuPage.js

```diff
@@ -70,23 +68,23 @@ const MenuPage = () => {
         <div className="menu-section">
           <div className="section-header">
             <h2 className="section-title">Hot Table</h2>
-            <p className="section-description">
+            {/* <p className="section-description">
```

Comment:
you can delete these instead of commenting out
try not to leave any comments in the code at all